client/tests/input_selection/alias_outputs.rs

```diff
@@ -2084,3 +2084,211 @@ fn state_transition_required_but_state_controller_not_provided() {
         })
     ));
 }
+
+#[test]
+fn state_transition_but_state_controller_not_owned() {
+    let protocol_parameters = protocol_parameters();
+    let alias_id_1 = AliasId::from_str(ALIAS_ID_1).unwrap();
+
+    let inputs = build_inputs(vec![Alias(
+        2_000_000,
+        alias_id_1,
+        0,
+        BECH32_ADDRESS_ED25519_0,
+        BECH32_ADDRESS_ED25519_1,
+        None,
+        None,
+        None,
+        None,
+    )]);
+    let outputs = build_outputs(vec![Alias(
+        2_000_000,
+        alias_id_1,
+        1,
+        BECH32_ADDRESS_ED25519_0,
+        BECH32_ADDRESS_ED25519_1,
+        None,
+        None,
+        None,
+        None,
+    )]);
+
+    let selected = InputSelection::new(
+        inputs,
+        outputs,
+        addresses(vec![BECH32_ADDRESS_ED25519_2]),
```

Comment:
```suggestion
        addresses(vec![BECH32_ADDRESS_ED25519_1]),
```
This might be even better, because it then also makes sure that we don't somewhere mix the governor as state controller

Reply:
Done